src/components/prefab-screens.js

```diff
@@ -22,136 +25,174 @@ const closeAfter = (f) => {
   }
   window.close();
 };
+const sendToApp = (customElement, command = "") => {
+  customElement.dispatchEvent(
+    new CustomEvent("requestMessage", {
+      bubbles: true,
+      detail: command,
+    })
+  );
+};
 
 const defineMessageScreen = (
-  tag,
-  img,
-  heading,
-  bodyText,
-  primaryAction,
-  onPrimaryAction,
-  secondaryAction = tr("getHelp"),
-  onSecondaryAction = () => closeAfter(() => open(sumoLink)),
-  totalPages = 0,
-  currentPage = 0
+  args = {
```

Comment:
Making this an arg obj as otherwise i have no clue what the order of the arguments is anymore after 4 weeks